src/components/insert_blog/InsertBlogComponent.tsx

```diff
@@ -0,0 +1,36 @@
+import { useState } from "react";
+import "./InsertStyle.css"
```

Comment:
import order, the order should be 
- 3rd parties
  + ReactJS related
  + Others
- Self-defined services/constants/… (alphabetically)
- Self-defined components (alphabetically)
- Assets file (images, css)